modules/programs/firefox.nix

```diff
@@ -33,6 +33,11 @@ in
         default = false;
         description = "Whether to enable the unfree Google Talk plugin.";
       };
+      enableIcedtea = mkOption {
```

Comment:
Can call this `enableIcedTea` to match the [official capitalization](https://icedtea.classpath.org/wiki/Main_Page).